src/bls12_381/fq.rs

```diff
@@ -21,6 +21,12 @@ const R2: FqRepr = FqRepr([0xf4df1f341c341746, 0xa76e6a609d104f1, 0x8de5476c4c95
 // INV = -(q^{-1} mod q) mod q
 const INV: u64 = 0x89f3fffcfffcfffd;
 
+// C1 = sqrt(-3) mod q = 1586958781458431025242759403266842894121773480562120986020912974854563298150952611241517463240701
+pub const C1: Fq = Fq(FqRepr([0x1dec6c36f3181f22, 0xb4b9bb641054b457, 0x25695a2be9415286, 0x982b6cbf66c749bc, 0x7d58e1ae1feb7873, 0x62c96300937c0b9]));
+
+// C2 = (C1 - 1) / 2 mod q = 793479390729215512621379701633421447060886740281060493010456487427281649075476305620758731620350
+pub const C2: Fq = Fq(FqRepr([0x30f1361b798a64e8, 0xf3b8ddab7ece5a2a, 0x16a8ca3ac61577f7, 0xc26a2ff874fd029b, 0x3636b76660701c6e, 0x51ba4ab241b6160]));
+
 // GENERATOR = 2 (multiplicative generator of q-1 order, that is also quadratic nonresidue)
 const GENERATOR: FqRepr = FqRepr([0x321300000006554f, 0xb93c0018d6c40005, 0x57605e0db0ddbb51, 0x8b256521ed1f9bcb, 0x6cf28d7901622c03, 0x11ebab9dbb81e28c]);
```

Comment:
It's 2 in the field. (Represented in Montgomery form, which is why it looks random.)